### PR TITLE
Optimizations and Beautifications

### PR DESCRIPTION
**Pull request by Mcray3000**
Streamlined some of the logic in our stores.py endpoint. Everything is handled, sorted, and messed with on the database side. Store hours are now just using the Python thing to get AM PM. Catalogs now returns food_id instead of SKU and shows price in dollars instead of cents.

### CODE REVIEW

**Review by kevinrutledge — APPROVED** [review]
Nothing to add. Your code improves the use cases. Good job.

File: src/api/stores.py [hunk 1]
```diff
@@ -36,138 +36,129 @@ def get_stores():
...
+                detail="Store does not found :(")
+        
+        catalog = conn.execute(fetch_catalog, {"store_id": store_id})
+
+    return_list = [
+        {
+        "food_id": item.food_id,
+        "item": item.name,
+        "quantity": item.quantity,
+        "price": f"${item.price / 100:.2f}"
```
Comment (TLMatthies) [inline src/api/stores.py]: Just a nitpick, but would prefer these to be indented one more to see that they're a separate object than the rest of stuff.

File: src/api/stores.py [hunk 2]
```diff
@@ -36,138 +36,129 @@ def get_stores():
...
-    for line in result:
-        return_object.append({
-            "store_id": line.id,
-            "store_name": line.name,
-            "price": line.price,
-            "rank": place
-        })
-        place += 1
+    with db.engine.connect().execution_options(isolation_level="REPEATABLE READ") as conn:
+        with conn.begin():
```
Comment (TLMatthies) [inline src/api/stores.py:140]: I don't think the second with statement is necessary here. I deleted it in testing and it still worked, and still threw the error when it was supposed to. What were you thinking the purpose of it was?

**Review by TLMatthies — APPROVED** [review]
It works, and it works well. Nice and readable. Thanks!

**Comment by TLMatthies** [comment]
Also, the times has been in the back of my head for a while. Thank you for fixing that!

File: src/api/stores.py (hunk 2 above)
Comment (Mcray3000) [reply src/api/stores.py:140]: Im unsure that part wasn't me

**Review by Aiden-Rodriguez — APPROVED** [review]
Nothing to add, absolutely epic work

**Pull request merged**